tomviz/modules/Module.h

```diff
@@ -114,7 +114,9 @@ class Module : public QObject
 
   /// Returns the active scalars of the module
   int activeScalars() const { return m_activeScalars; }
-  static const int DEFAULT_SCALARS;
+  QString activeScalarsName() const;
+  static const QString DEFAULT_SCALARS_NAME;
```

Comment:
Can we not use macro style names, I think using s_defaultScalarsName and s_defaultScalarsIdx would be preferable.